src/HTSPConnection.cpp

```diff
@@ -362,7 +394,6 @@ htsmsg_t *CHTSPConnection::SendAndWait0 ( const char *method, htsmsg_t *msg, int
   if (!htsmsg_get_u32(msg, "noaccess", &noaccess) && noaccess)
   {
     // access denied
-    //XBMC->QueueNotification(QUEUE_ERROR, "Command %s failed: Access denied", method);
```

Comment:
No, as this is not a connection related error it does not alter the connection state. It's just a single operation (e.g. delete recording or timer) that failed.